### ISSUE TITLE
What about making supportsTablet's default value true

### ISSUE THREAD

**Issue by btk** [expo/expo#1456]
Hey, today after uploading my binary to appstore, saw that the build does not support tablet. Then discovered that  `supportsTablet` property needs to be defined as true. 

Since most of us developers use the flex logic and do our work responsive, isn't it better to have it come true as default? 

Or if I'm missing something what is the rationale behind making it false in default? Thank you!

**Comment by terribleben** [comment]
Hi there!

We do have true as the [default in our starter project](https://github.com/expo/new-project-template/blob/b19be14991a776e4173801c7602bfec586901041/app.json#L18).

For people who omit the key, we aren't sure whether they've [tested on an iPad](https://docs.expo.io/versions/latest/guides/app-stores.html#apple-can-reject-your-app-if-elements-dont-render-properly-on-an-ipad-even-if-your-app-doesnt-target-the-ipad-form-factor-be-sure-and-test-your-app-on-an-ipad-or-ipad-simulator) and don't want to do something they don't expect, so we just do the same thing as a blank native iOS project. Unfortunately it seems like Apple can be picky about this either way.

Hope that answer is helpful!

**Comment by PrimeTimeTran** [comment]
@terribleben question about the AppStore deployment process, if we set this to be false because our application does not support Tablet, does the AppStore still require us "design for tablet"? 

I got a rejection because I never styled for Tablet.

My application doesn't need to be available on Tablet for now so I'd rather not go back through styling everywhere if I don't have to. 

Thanks for your help in advance~!
